cli/README.md

```diff
@@ -554,6 +555,12 @@ This specifies the relative path to where the generated JS bundle's sourcemap fi
 
 *NOTE: This parameter can be set using either --sourcemapOutput or -s*
 
+#### Output directory parameter
+
+This specifies the relative path to where the assets, JS bundle and sourcemap files should be written. If left unspecified, assets, JS bundle and sourcemap will copied to /tmp/CodePush folder by default.
```

Comment:
Suggested modifications:
- will be copied
- to the `/tmp/CodePush` folder
- Could you put back-ticks around the directory path?